plonky2x/core/src/backend/circuit/build.rs

```diff
@@ -40,10 +40,9 @@ impl<L: PlonkParameters<D>, const D: usize> CircuitBuild<L, D> {
         PublicInput::new(&self.io)
     }
 
-    /// Generates a proof for the circuit. The proof can be verified using `verify`.
-    pub fn prove(
+    pub fn prove_with_partial_witness(
         &self,
-        input: &PublicInput<L, D>,
+        inputs: PartialWitness<L::Field>,
```

Comment:
can you call this 'partial_witness', I think the variable naming is a bit incorrect